React/Base/RCTDefines.h

```diff
@@ -69,9 +69,11 @@
 #define RCT_IF_DEV(...)
 #endif
 
+#if !TARGET_OS_OSX // TODO [GH #533]
```

Comment:
Probably worth a comment that we should turn this off on iOS as well so we're not doing this at all in app store builds that consume this?